tasks/connectors/edgescan/lib/edgescan_api.rb

```diff
@@ -74,6 +76,18 @@ def base_url
 
           "https://#{@api_host}"
         end
+
+        # Application vulnerabilities are on layer 7
+        # Network vulnerabilities are on every other layer
+        def layer_query_parameters
+          if @include_application_vulnerabilities && @include_network_vulnerabilities
```

Comment:
What happens if a user disables the import of both application layer and network layer vulnerabilities?

Reply:
Fixed in commit dd27498.
If a user disables both application and network layer vulnerabilities then no vulnerabilities will be imported, but assets will still be created.